Return to main menu

GrzegorzDryja opened the pull request:
Added return to main menu function from program class after saving note.
SaveFile function is not static anymore

Commit:
Return to main menu

Added return to main menu function from program class after saving note.
SaveFile function is not static anymore

tomkowy commented on Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs:
```diff
@@ -6,7 +6,7 @@ namespace Notatnik_parser.Notepad
     public class NotepadMenu
     {           
-        static void saveFile(ref string textNoteParameter)
+        public void SaveFile(ref string textNoteParameter)
```
Niestety nie zauważyłem ref'a w poprzednim a'la review. Nie potrzebujesz go używać. Metoda jest dalej publiczna, czemu?

tomku-mar reviewed (changes requested):
Jeszcze parę drobnych zmian

Commit:
Update SaveFile function

Global use of SaveFile function was on my mind. Now SaveFile function is set to private.
"ref" is mentioned in official MS guide - https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/passing-parameters
Code works now without ref key word.
